### PR TITLE
fix gitlab_project avatar_path open when undefined bug (#3926)

### PR DESCRIPTION
**Pull request by josham**
##### SUMMARY
Any call to gitlab_project to create with avatar_path undefined will throw an exception

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
gitlab_project

### CODE REVIEW

File: changelogs/fragments/3926-fix-gitlab-project-avatar-path.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - gitlab - gitlab_project needs to check for avatar_path before trying to open (https://github.com/ansible-collections/community.general/issues/3926).
```
Comment (felixfontein) [inline changelogs/fragments/3926-fix-gitlab-project-avatar-path.yml:2]: I just saw that `avatar_path` hasn't been in a release yet, so this doesn't need a changelog fragment. Please remove this file.
```suggestion
```

**Comment by felixfontein** [comment]
The same bug also exists in gitlab_group. Would you mind fixing it there as well?

**Comment by josham** [comment]
This bug does not exist in gitlab_group

**Comment by felixfontein** [comment]
Ah right, I just saw I looked wrong :)

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@josham thanks for fixing this!

**Pull request merged**